mkosi/__init__.py

```diff
@@ -1449,6 +1449,7 @@ def print_summary(config: MkosiConfig) -> None:
         With Documentation: {yes_no(config.with_docs)}
              Package Cache: {none_to_none(config.cache_dir)}
                Extra Trees: {line_join_source_target_list(config.extra_trees)}
+            Skeleton Trees: {line_join_source_target_list(config.skeleton_trees)}
```

Comment:
Somehow I missed that this was removed. While that's the order it was printed in before, mentioning skeleton trees before extra trees would make a bit more sense, since they are applied first.
```suggestion
            Skeleton Trees: {line_join_source_target_list(config.skeleton_trees)}
               Extra Trees: {line_join_source_target_list(config.extra_trees)}
```

Reply:
Done.

Initially, I also wanted to check the order of the summary, but gave up and just put back skeleton trees where it was.